### DISCUSSION TITLE
Conservation areas

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#33] [Planning consideration]
# Conservations areas

A conservation area is an area of notable architectural or historical interest which is protected by law against undesirable changes - such as development that will change the overall aesthetics of the area.

## What

Conservation areas are designated by Local Planning Authorities and are usually visible on their website (for example, [the conservation areas in Leeds](https://www.leeds.gov.uk/planning/conservation-protection-and-heritage/conservation-area)).

We developed a [Standard for conservation areas](https://digital-land.github.io/specification/specification/conservation-area/) so that Local Authorities could start publishing their list of conservation areas as data, we could aggregate a national dataset of conservation areas, and this dataset could be used by data consumers (such as PropTech companies).

The standard currently models a record of the conservation, including a geometry and any supporting documents that evidence the legality of the area. It does not include a breakdown of the restrictions - this is something we might consider in a future iteration of the standard.

The planning data platform currently collects conservation areas from 62 Local planning authorities and has [a dataset containing ~8600 records](https://www.planning.data.gov.uk/dataset/conservation-area).

## Why

Development in conservation areas is usually restricted. Therefore, it is important anyone attempting to do some development can find out if they are in a conservation area and if so, what restrictions apply.

Any software being built to help applicants would need this data to be able to perform these checks for users.

## What we're looking for

We would like to know:

* if the data matching the current standard is useful and how it is being used?
* if you think the initial standard captures enough useful information as data
* what documents should be recorded as data
* how you'd like to or currently do use conservation data
* other than the data available on the platform, do you know if other conservation area exists
* anything else you think we should know

**Comment by mattlucht** [comment]
We're proposing to make a change to the [conservation area specification](https://digital-land.github.io/specification/specification/conservation-area/). If/when we make these changes we'll ensure that any data that has already been provided is backward compatible.

*  we're proposing to add `point` to the `conservation-area`. If a `geometry` has already been provided then we'll automatically determine the `point` from the mid-point

**Reply by colmjude** [reply]
We've made this change now and bumped the version to [v1.2.2](https://digital-land.github.io/specification/specification/conservation-area/v1.2.2/)

**Comment by mattlucht** [comment]
At the moment we're using `start-date` to define when a conservation area became a conservation area. In line with how we're defining new specifications we'd like to use a more contextual attribute, but we're not sure how to describe it – would `designation-date` be the right thing to use?

**Reply by colmjude** [reply]
The [NPPF](https://www.gov.uk/guidance/national-planning-policy-framework/16-conserving-and-enhancing-the-historic-environment) refers to the "designation of conservation areas" so `designation-date` sounds right.

**Reply by colmjude** [reply]
We have now added `designation-date` to the [conservation-area standard](https://digital-land.github.io/specification/specification/conservation-area/#conservation-area-dataset).

As part of our drive to reduce ambiguity in our standards we recognise that one area in which we have introduced ambiguity is with dates. 

Previously we used the [common dates](https://standards.planning-data.dev/rules/#all-schemas-must-include-entry%2C-start-and-end-dates), in particular, `start-date` to mean different things in different datasets. This was confusing. 

We are trying to reduce the chance to misinterpret the purpose of a field by using [more meaningful names](https://standards.planning-data.dev/patterns/contextual-dates/). If it is an important date to record it should get its own appropriately named field.

That is why we've added `designation-date` to the conservation-area dataset. We chose `designation-date` to mirror the language used in [the legislation describing how a conservation area is designated](https://www.legislation.gov.uk/ukpga/1990/9/section/69).

Please let us know your thoughts on this change and if there are any other ambiguous items in the standards.

**Reply by psd** [reply]
I am concerned about this decision which makes things harder for users of the data, and is [empirically leading to confusing data](https://datasette.planning.data.gov.uk/conservation-area?sql=select+entity%2C+name%2C+start_date%2C+json_extract%28json%2C+%27%24.designation-date%27%29+as+dd+from+entity+where+dd+%21%3D+start_date+and+dd+%21%3D+%27%27) in practice. 

Our data architecture uses a common `start-date` field to indicate when the entity comes into effect. In this case we now have records where the optional `designation-date` differs from the mandatory `start-date`. 

My understanding is the date of designation is the date the controls take effect, so is the start-date. What does that actually mean to have two different dates for a user? Which is the date from which the conservation area needs to be considered for planning applications?

We will seek legal advice to clarify if the designation is ever different to the application data. If they are the same, then we could document `designation-date` in the specification as being a synonym for `start-date`, though we will need to be clear which field takes precedence. And that one of the values will be ignored if two fields with different values are provided.

Whilst I can see using meaningful names can help users provide data, it's a pattern which needs more thought before using it in our specifications.

**Comment by colmjude** [comment]
We are going to add `document-url` to the [conservation-area dataset](https://digital-land.github.io/specification/specification/conservation-area/#conservation-area-dataset).

It has come up in the data drop-in sessions that some LPAs think it would be beneficial to have a `document-url` field where data providers can add a URL that links directly to the authoritative source of the area (often a pdf on the area drawn on a map).

We agree with the rationale.

It increases the value of the [conservation-area dataset](https://digital-land.github.io/specification/specification/conservation-area/#conservation-area-dataset) because from each record you can always get to the authoritative source for the area. It is explicit about what is the authoritative source of the area, rather than looking through all documents recorded for a conservation area and choosing what could be the authoritative source from those that are available.

This will build trust in the data because wherever the conservation-area data is used, the service provider, whether that is [planning.data.gov.uk](http://planning.data.gov.uk/), PlanX, or any other service provider, will be able to expose a link directly to the authoritative, legally binding source for the area.

In terms of providing this data:

* Providers that have URLs to these documents, whether HTML, PDF or other, readily available can start adding them now.
* Providers that don't have these available yet can add it once they have it available.

As always, gaps in your data are okay. We would rather you leave a field blank if you don’t know or have anything to put there than include something you don’t think is right. Gaps can be filled in later where possible

**Reply by colmjude** [reply]
We've made this change now.

Useful links:

* [v1.4.3 of the specification](https://digital-land.github.io/specification/specification/conservation-area/)
* [Updated guidance](https://www.planning.data.gov.uk/guidance/specifications/conservation-area)

**Reply by trodaway** [reply]
I've asked this [question on Slack](https://opendigitalplanning.slack.com/archives/C05NQJK5S9L/p1737134848900909), but I'll ask it here too: Given we have the area maps URL added as a dedicated field in the related Conservation Area dataset, should we be including these as entries in the Conservation Area Document dataset too?

### Edit
This has now been [answered on Slack](https://opendigitalplanning.slack.com/archives/C05NQJK5S9L/p1737628005453839?thread_ts=1737134848.900909&cid=C05NQJK5S9L) by @kieranwint:
> No need to duplicate document in conservation area document. You can put the doc with red line boundary in `conservation-area` dataset. No need to also add to `conservation-area-document`

**Comment by colmjude** [comment]
## Moving to candidate standard

We have been piloting [this data standard](https://digital-land.github.io/specification/specification/conservation-area/) for a while with a small number of data providers.

What we’ve learnt during this phase has helped us to tweak and improve the data specification - specifically [adding a couple of extra fields](https://digital-land.github.io/specification/specification/conservation-area/changelog) that we as a community think improve the legibility and increases the value of the datasets.

Analysis of the data produced to this standard so far suggests the standard is working for the vast majority of data providers.

Most of the providers who have provided [conservation-area data](https://www.planning.data.gov.uk/dataset/conservation-area) have included data for the fields we would expect - often `end-date` and `note` are empty but that is to be expected.

It is a similar story for the [conservation-area-document dataset](https://www.planning.data.gov.uk/dataset/conservation-area-document). Although fewer providers have provided this dataset, those that have provided data have done so to a high level of completeness.

We are going to move the conservation area standard from “pilot” standard to “candidate” standard.

This means we’re confident that the standard is fit for purpose, works for providers and produces valuable data.

The standard may still expand and evolve but we don’t expect to have to drastically overhaul it.

During the candidate stage, the standard will be tested with a larger group of data providers and reviewed by our legal teams before being worked into legislation as part of the [Levelling Up and Regeneration Act 2023](https://www.legislation.gov.uk/ukpga/2023/55/contents/enacted).

Any questions, please do ask

**Comment by kieranwint** [comment]
## OS Update

The last update we provided on Conservation Areas was that we were progressing the standard from “pilot” to “candidate.” 

Since then, we have been working with Ordnance Survey to gather a statement to help increase confidence in how to create the geometries within the specifications.

The OS have confirmed that should a geometry be a conservation area that is listed under the Listed Buildings and Conservation Areas Act 1990, it will fall under the presumption to publish guidelines, but do consult their website [here](https://www.ordnancesurvey.co.uk/customers/public-sector/public-sector-licensing/publish-derived-data), under the “planning” sub-heading.

If the geometries included in your conservation area dataset is produced in line with our technical specifications they can be shared under the open government licence.

If you have any thoughts, questions or concerns, please do let us know.

**Comment by stevenjmesser** [comment]
Leicester thinks a new document type for management plans should be added, as they're very different to area appraisals and notices. Originally raised [in Slack (requires a login)](https://communities-govuk.slack.com/archives/C05NQJK5S9L/p1728641383278619).

**Reply by colmjude** [reply]
Thanks @stevenjmesser 
We've added a `management-plan` to the [list of document types](https://dluhc-datasets.planning-data.dev/dataset/conservation-area-document-type/finder).

If we are able to get a description of management plans we can add that to help users to understand the differences between management plans and the other document types.

**Reply by stevenjmesser** [reply]
Is there anything in section 6 of [Historic England's Conservation Area Appraisal, Designation and Management publication](https://historicengland.org.uk/images-books/publications/conservation-area-appraisal-designation-management-advice-note-1/heag-268-conservation-area-appraisal-designation-management/)?

**Comment by dkb-tpx-00** [comment]
Two issues:
1)   I think clarification around whether a Character Appraisal is an Appraisal?
Does a conservation area have to be in place to have a Character Appraisal on it? 
Some LAs seem to use the two as interchangeable,
What about Management plans, as some local authorities do not appraisals as such but do have character appraisals and/or management plans. Would it be useful to have a field to denote the document type?
2)   start-date: Is this Adoption, Designation or Endorsed?

**Reply by dkb-tpx-00** [reply]
Just noticed that we have document type, but perhaps types need adding to?
Also I am not denoting document type in the record sets I am creating; https://docs.google.com/spreadsheets/d/1r96ruDklYhgm0m0mASR2Bsxc7KE59FQ1

**Reply by colmjude** [reply]
Hi @dkb-tpx-00 - thanks for your comments.

Yes there is a `document-type` field on the conservation-area-document dataset. The types we are looking for are still emerging so if you have an ideas about types of conservation area documents we should be collecting please do let us know. If you can give us a description of the document type and what it is used for that would be great.

The 5 we have at the moment are listed [here ](https://dluhc-datasets.planning-data.dev/dataset/conservation-area-document-type/finder)and are the most common types we've received from planning authorities so far.

In terms of your question about `start-date` it would be great to know more about Adoption, Designation or Endorsed, we've not come across these terms in relation to conservation areas before. Are these statuses for a conservation area or something to do with specific conservation area documents?

**Reply by dkb-tpx-00** [reply]
Hi there,
I would suggest adding "Character Statement" as a document-type. For example Brighton and Hove City Council, https://www.brighton-hove.gov.uk/planning/heritage/documents-avenues-conservation-area.

Adoption, Designation and Endorsed are just the definition of dates used by planning authorities. I am assuming at present that they all mean **started**!

**Reply by dkb-tpx-00** [reply]
From https://www.huntingdonshire.gov.uk/planning/conservation-areas

**Character statements**
We produce character statements for the conservation areas in our district. The statements describe the features, history and development of the conservation area. They are a useful guide to the type of new development that could be considered to make a positive contribution to local character and distinctiveness. The character statement should be referred to in any Heritage Statement submitted as part of a planning application if the site is located within a conservation area.

**Reply by dkb-tpx-00** [reply]
Oooo, there is a "**Design Statement**" too!

https://www.huntingdonshire.gov.uk/media/2327/hilton-village-design-statement-colour.pdf

**Reply by trodaway** [reply]
We similarly have a number of "Design guides", so something suitable for those to be included could prove useful.

**Reply by colmjude** [reply]
We have added "Design guidance" to [the list of document types](https://dluhc-datasets.planning-data.dev/dataset/conservation-area-document-type/finder) because a few people have pointed this document type out and it is explicitly called out on page 32 of [Historic England's Conservation Area Appraisal, Designation and Management publication](https://historicengland.org.uk/images-books/publications/conservation-area-appraisal-designation-management-advice-note-1/heag-268-conservation-area-appraisal-designation-management/)

**Comment by trodaway** [comment]
We have some documents that are published as a single PDF but containing two parts, so would meet multiple document types. How do we handle this? Do we duplicate the document as an entry, or is there an order of precendence we should be selecting them from where multiple? A lot of the character appraisals mention management matters, but only briefly, however our documents for [Berwick](https://www.northumberland.gov.uk/NorthumberlandCountyCouncil/media/Planning-and-Building/Conservation/Berwick-CACA.pdf), [Spittal](https://www.northumberland.gov.uk/NorthumberlandCountyCouncil/media/Planning-and-Building/Conservation/Spittal-CACA.pdf) and [Tweedmouth](https://www.northumberland.gov.uk/NorthumberlandCountyCouncil/media/Planning-and-Building/Conservation/Tweedmouth-CACA.pdf) contains a full management strategy too. An example of our appraisals that more briefly mentions management matters is [Whitton](https://www.northumberland.gov.uk/NorthumberlandCountyCouncil/media/Planning-and-Building/Conservation/Whitton-CAA.pdf) - these are typically titled as 'Character Appraisal and Management Matters' and contain a handful of pages on 'Designation & Management Matters'.

p.s. Also [asked on Slack](https://opendigitalplanning.slack.com/archives/C05NQJK5S9L/p1737134848900909)


### Edit
This has now been [answered on Slack](https://opendigitalplanning.slack.com/archives/C05NQJK5S9L/p1737628005453839?thread_ts=1737134848.900909&cid=C05NQJK5S9L) by @kieranwint:
> If single PDF has 2 docs, put 2 rows in the dataset. `document-url` will be the same for both rows but you can put a different `document-type`. You can also use the `notes` to note which pages of pdf are for the document (e.g. Character Appraisal and Management Matters is pages 5-10).
